scrapers/TheNude.yml

```diff
@@ -78,7 +79,13 @@ xPathScrapers:
       Piercings: 
         selector: //li/span[@class="list-quest"][contains(text(),'Piercings')]/../text()
       Image:
-        selector: (//meta[@itemprop="image"])[1]/@content
+        selector: //meta[@itemprop="url"]/@content
```

Comment:
the selector as mentioned above gets 2 results
using the `(//meta[@itemprop="url"]/@content)[1]` instead should get us only the first

Reply:
This suggestion here also breaks the image selection.

Reply:
damn.... idk why it happens. for me it seems to work but not all of the times...